index.js

```diff
@@ -12,6 +12,10 @@ function dbfRowHeader(buffer){
 	var data = new DataView(buffer);
 	var out = [];
 	var offset = 32;
+	var testByte = new Uint8Array(buffer,offset,1);
```

Comment:
I was just mimicking the next action on the buffer, ```new Uint8Array(buffer,offset,11)```

I haven't used DataViews before - is there a risk that I could somehow consume the first byte and remove it from the header row?